modules/control-utility/src/test/java/org/apache/ignite/util/GridCommandHandlerConsistencyTest.java

```diff
@@ -138,6 +148,26 @@ private void testAtomicAndTx(boolean incVal) throws Exception {
 
         injectTestSystemOut();
 
+        LogListener lsnrUnmaskedKey = matches("Key: 0 (cache: ").build(); // Unmasked key.
+        LogListener lsnrMaskedKey = matches("Key: [HIDDEN_KEY#").build(); // Masked key.
+        LogListener lsnrMaskedVal = matches("Value: [HIDDEN_VALUE#").build(); // Masked value.
+
+        listeningLog.registerListener(lsnrUnmaskedKey);
+        listeningLog.registerListener(lsnrMaskedKey);
+        listeningLog.registerListener(lsnrMaskedVal);
+
+        List<LogListener> listeners = new ArrayList<>();
+
+        // It's unable to check just "Key:" count while https://issues.apache.org/jira/browse/IGNITE-15316 not fixed
+        if (S.includeSensitive())
```

Comment:
Square brackets required

Reply:
Fix it here and in another places, also.

Reply:
Done